### PR TITLE
Tutorial reviewed and changed for better understanding

### PR DESCRIPTION
**Pull request by kathyguerron**
@acamino ready! @asirerac

### CODE REVIEW

File: tutorial/index.jade [hunk 1]
```diff
@@ -95,7 +97,7 @@
     :markdown
       ## Registration server side implementation
```
Comment (acamino) [inline tutorial/index.jade]: Should this title be capitalized?

Comment (kathyguerron) [inline tutorial/index.jade]: Yes it should, good catch

File: tutorial/index.jade [hunk 2]
```diff
@@ -152,25 +155,25 @@
...
     :markdown
       ## Sending the Confirmation Message
 
-      Much as we did for our Authy client, we create a single instance of the
-      Twilio REST API helper, called `twilio_client` in this example.
+      Just as we did for our Authy client, we create a single instance of the
+      Twilio REST API helper. It will be called `twilio_client` in this example.
 
-      Then all we need to do to send an sms to the user's phone is use the built
+      Then all we need to do is send an sms to the user's phone that was used
```
Comment (acamino) [inline tutorial/index.jade]: Should _sms_ be capitalised?

File: tutorial/index.jade [hunk 3]
```diff
@@ -46,9 +47,10 @@
 
       SQLALCHEMY_DATABASE_URI = 'sqlite://'
       ```
+
       Note that you have to replace the placeholders `your_twilio_account_sid`, `your_twilio_auth_token`, `your_twilio_phone_number`
-      and `your_authy_key` with your production information,
-      which you can find in your [Twlio Account Dashboard](/user/account) and your [Authy
+      and `your_authy_key` with your production information.
+      You can find it in your [Twlio Account Dashboard](/user/account) and your [Authy
```
Comment (acamino) [inline tutorial/index.jade]: Could you please fix the typo **Twlio**?

**Comment by acamino** [comment]
@kathyguerron I left a couple of comments, would you mind addressing them?

I find _sign-up_ more natural than _signup_, what do you think?

> 1. The users begin the registration process by entering their data, including a phone number, into a sign-up form.

**Comment by acamino** [comment]
Thanks @kathyguerron!

File: tutorial/index.jade [hunk 4]
```diff
@@ -152,25 +155,25 @@
...
     :markdown
       ## Sending the Confirmation Message
 
-      Much as we did for our Authy client, we create a single instance of the
-      Twilio REST API helper, called `twilio_client` in this example.
+      Just as we did for our Authy client, we create a single instance of the
+      Twilio REST API helper. It will be called `twilio_client` in this example.
 
-      Then all we need to do to send an sms to the user's phone is use the built
+      Then all we need to do is send an SMS to the user's phone that was used
```
Comment (jsantos17) [inline tutorial/index.jade]: "that was" makes this sentence clumsy. "Then all we need to do is send an SMS to the phone number used in "messages.create()" is much better.

File: tutorial/index.jade [hunk 5]
```diff
@@ -136,13 +139,13 @@
...
-      to that user's mobile phone. 
+      Once the user has an `authyId` we can actually send a verification code
+      to that user's mobile phone.
 
-      When our user is created successfully via the form we implemented, we send
+      When our user is created successfully, via the form we implemented, we send
       a token to the user's mobile phone to verify their account in our
-      controller. When the code is sent, we redirect to another page where the
-      user can enter the token they received, completing the verification process.
+      controller. When the code is sent, we redirect him to another page where
```
Comment (jsantos17) [inline tutorial/index.jade]: Not a problem or anything but we usually use the [singular, gender-neutral they](http://www.americandialect.org/2015-word-of-the-year-is-singular-they) or default to she/her.

**Pull request merged**